static/app/views/settings/organizationAuthTokens/index.spec.tsx

```diff
@@ -0,0 +1,288 @@
+        expect.objectContaining({method: 'GET', query: {query: `id:${project.id}`}})
+      );
+    });
+  });
+
+  it('handle error when loading tokens', async function () {
+    const mock = MockApiClient.addMockResponse({
+      url: ENDPOINT,
+      method: 'GET',
+      statusCode: 400,
+    });
+
+    render(<OrganizationAuthTokensIndex {...defaultProps} />);
+
+    await waitFor(() => {
+      expect(mock).toHaveBeenCalledTimes(1);
+    });
+
+    expect(await screen.findByTestId('loading-error')).toHaveTextContent(
+      'Failed to load auth tokens for the organization.'
+    );
+    expect(screen.queryByTestId('loading-indicator')).not.toBeInTheDocument();
+    expect(screen.queryByTestId('empty-state')).not.toBeInTheDocument();
+  });
+
+  it('shows empty state', async function () {
+    const tokens: OrgAuthToken[] = [];
+
+    MockApiClient.addMockResponse({
+      url: ENDPOINT,
+      method: 'GET',
+      body: tokens,
+    });
+
+    render(<OrganizationAuthTokensIndex {...defaultProps} />);
+
+    // First shows loading indicator
+    await screen.findByTestId('loading-indicator');
+
+    // Then empty state
+    expect(await screen.findByTestId('empty-state')).toBeInTheDocument();
+    expect(screen.queryByTestId('loading-error')).not.toBeInTheDocument();
+    expect(screen.queryByTestId('loading-indicator')).not.toBeInTheDocument();
+  });
+
+  describe('revoking', function () {
+    it('allows to revoke tokens', async function () {
+      jest.spyOn(indicators, 'addSuccessMessage');
+
+      const tokens: OrgAuthToken[] = [
+        {
+          id: '1',
+          name: 'My Token 1',
+          tokenLastCharacters: '1234',
+          dateCreated: new Date('2023-01-01T00:00:00.000Z'),
+          scopes: ['org:read'],
+        },
+        {
+          id: '2',
+          name: 'My Token 2',
+          tokenLastCharacters: 'ABCD',
+          dateCreated: new Date('2023-01-01T00:00:00.000Z'),
+          scopes: ['org:read'],
+        },
+        {
+          id: '3',
+          name: 'My Token 3',
+          tokenLastCharacters: 'ABCD',
+          dateCreated: new Date('2023-01-01T00:00:00.000Z'),
+          scopes: ['org:read'],
+        },
+      ];
+
+      MockApiClient.addMockResponse({
+        url: ENDPOINT,
+        method: 'GET',
+        body: tokens,
+      });
+
+      const deleteMock = MockApiClient.addMockResponse({
+        url: `${ENDPOINT}2/`,
+        method: 'DELETE',
+      });
+
+      render(<OrganizationAuthTokensIndex {...defaultProps} />);
+      renderGlobalModal();
+
+      await screen.findByText('My Token 1');
+      screen.getByText('My Token 2');
+      screen.getByText('My Token 3');
+
+      expect(screen.getByLabelText('Revoke My Token 2')).toBeEnabled();
+
+      await screen.getByLabelText('Revoke My Token 2').click();
+      // Confirm modal
+      await userEvent.click(screen.getByRole('button', {name: 'Confirm'}));
+
+      await screen.findByText('My Token 1');
```

Comment:
it sounds repetitive and not really necessary but we agreed on one of the past TSC meetings that we should be explicit and always use the `expect....toBeInTheDocument`

Reply:
I've updated everything according to this!